content/en/real_user_monitoring/flutter/mobile_vitals.md

```diff
@@ -18,17 +18,25 @@ further_reading:
 
 Real User Monitoring offers Mobile Vitals, a set of metrics that can help compute insights about your mobile application's responsiveness, stability, and resource consumption. Mobile Vitals range from poor, moderate, to good.
 
-You can view Mobile Vitals for your application by navigating to  **UX Monitoring > Performance Summary** and selecting your application.
+You can see a high level overview of Mobile Vitals for your application by navigating to  **UX Monitoring > Performance Summary** and selecting your application.
 
 {{< img src="real_user_monitoring/flutter/flutter-mobile-vitals-1.png" alt="Mobile Vitals in the Performance Summary tab" style="width:90%;">}}
 
 To access the RUM mobile app performance dashboard, scroll down to the Monitor Performance Metrics section from the application summary page and click **Performance Overview**.
 
 {{< img src="real_user_monitoring/flutter/flutter-performance-overview.png" alt="Mobile performance overview dashboard" style="width:90%;">}}
 
-Understand your application's overall health and performance with the line graphs displaying metrics across various application versions. To filter on application version or see specific sessions and views, click on a graph. 
+Understand your application's overall health and performance with the line graphs displaying metrics across various application versions. To filter on application version or see specific sessions and views, click on a graph.
 
-{{< img src="real_user_monitoring/ios/rum_explorer_mobile_vitals.png" alt="Event Timings and Mobile Vitals in the RUM Explorer" style="width:90%;">}}
+When viewing performance details by the **View** event type, the following event timing and mobile vitals information is available:
+
+- Refresh Rate Average
+- CPU Tickets Per Second
+- Memory Average
+- View Flutter Build TIme Average
+- Raster Time Average
+
+{{< img src="real_user_monitoring/flutter/rum_explorer_mobile_vitals-2.png" alt="Event Timings and Mobile Vitals in the RUM Explorer" style="width:90%;">}}
 
 You can also select a view in the RUM Explorer and observe recommended benchmark ranges that directly correlate to your application's user experience in the session. Click on a metric such as **Refresh Rate Average** and click **Search Views With Poor Performance** to apply a filter in your search query and examine additional views.
 
```

Comment:
for standardization, can we remove the section below this? and copy paste the same table we have for android, ios, or react native? and make sure we add widget build time and raster time in it (@fuzzybinary can advise if needed)

Reply:
@fuzzybinary would you mind filling in the blanks here?

Reply:
** Build Time Average **
This is the duration of time taken build the frame on the UI thread. To ensure a smooth animations, this should not exceed 16ms for 60 FPS, and 8ms for 120 FPS.  High values here mean you need to look into optimizing your `build` methods for this view. See [Control Build Cost](https://docs.flutter.dev/perf/best-practices#control-build-cost) in the Flutter documentation

** Raster Time Average **
This is the duration of time taken to rasterize the frame on the raster thread. To ensure a smooth animations, this should not exceed 16ms for 60 FPS, and 8ms for 120 FPS.  High values here may mean your view is complex to render. See [Identifying Problems in the GPU Graph](https://docs.flutter.dev/perf/ui-performance#identifying-problems-in-the-gpu-graph) in the Flutter documentation.